cli-commands/ganache/ganache.js

```diff
@@ -1,32 +1,36 @@
 const ganache = require('ganache-cli');
 const setup = require('./setup.json');
-const logger = require('./../../logger-service/logger-service').logger;
+const colors = require('./../../utils/colors');
+const logger = require('../../logger-service/logger-service').logger;
 let port;
 
-const run = (inPort) => {
+const run = (inPort, inLogger, forkParams) => {
 	port = (inPort) ? inPort : setup.defaultPort;
+	fork = (forkParams) ? forkParams : setup.forkParams;
 	const server = ganache.server({
 		accounts: setup.accounts,
-		logger
+		logger: inLogger,
+		fork: fork
 	});
 
 	server.listen(port, ganacheServerListenCallback);
 };
 
 const ganacheServerListenCallback = (err, blockchain) => {
-
 	if (err) {
-		console.log(err);
+		logger.log(err);
 		return;
 	}
-
 	const accountsLength = blockchain.options.accounts.length;
-
+	const forkedNetwork = blockchain.options.fork;
+	const forkedBlockNumber = parseInt(blockchain.options.fork_block_number, 16);
```

Comment:
Why parse it to number. Cant you use the string and just check if is != undefined and length > 0

Reply:
Because it is returned as hex number, for example 0x336546 and I want to show to user the block number in decimal format (human readable format).

Reply:
Got it! Great job